src/framework/mlt_frame.c

```diff
@@ -1228,3 +1228,128 @@ int mlt_image_format_planes( mlt_image_format format, int width, int height, voi
 
 	return 0;
 }
+
+/** Get the short name for a channel configuration.
+ *
+ * You do not need to deallocate the returned string.
+ * \public \member of mlt_frame_s
+ * \param cfg a channel configuration enum
+ * \return a string for the name of the channel configuration
+ */
+
+const char * mlt_chan_cfg_name( mlt_chan_cfg cfg )
```

Comment:
These are the strings that the user would use to specify the channel conifguration. E.g:

`melt clip.mp4 -consumer sdl channels=4 chan_cfg="quad(back)"`

Any comments on the names? We could replace the "()" with "_". For example: "quad_back".